### ISSUE TITLE
undefined is not an object if no metaCreator is given

### ISSUE THREAD

**Issue by skolkmann** [redux-utilities/redux-actions#311]
I used 2.2.1 which worked like a charm, now upgraded to 2.5.0 and get:

undefined is not an object (evaluating 'metaCreator.apply')

Happens in both createAction(s):
const actionCreator = createAction('ACTION') // Error
const actionCreator = createAction('ACTION', undefined, undefined) // Error
const actionCreator = createAction('ACTION', undefined, null) // Works
const actionCreator = createAction('ACTION', undefined, () => {}) // Works

**Comment by mewdriller** [comment]
This error just started cropping up for us as well. Our `yarn.lock` had us at v2.4.0 before and v2.5.0 after. So it looks like it's v2.5.0+ that's the issue.

**Comment by timche** [comment]
Thanks for confirming this. 

For everyone having this issue please keep ur version locked to < v2.50, sorry. 

I'll try to fix it asap, if someone is willing to help, feel free to let me know and open a PR.

**Comment by timche** [comment]
So I couldn't to replicate this issue.

```js
import createAction from '../src/createAction';

test('a', () => {
  const a = createAction('TYPE');
  expect(a()).toEqual({ type: 'TYPE' })
})

test('b', () => {
  const b = createAction('TYPE', undefined, undefined);
  expect(b()).toEqual({ type: 'TYPE' })
})

test('c', () => {
  const c = createAction('TYPE', null, undefined);
  expect(c()).toEqual({ type: 'TYPE' })
})

test('d', () => {
  const d = createAction('TYPE', undefined, null);
  expect(d()).toEqual({ type: 'TYPE' })
})
```

They all pass. What do I miss?

**Comment by timche** [comment]
I have released [v2.6.0](https://github.com/redux-utilities/redux-actions/releases/tag/v2.6.0) and [v2.6.1](https://github.com/redux-utilities/redux-actions/releases/tag/v2.6.1). Can you please check if the error is still there?

**Comment by skolkmann** [comment]
Is working now in 2.6.1, thanks for your fast response!

Just as side notice, I am using this in react-native.
